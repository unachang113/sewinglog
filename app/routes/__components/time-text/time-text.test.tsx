import {render} from '@testing-library/react';
import {TimeText} from '.';

describe('TimeText', () => {
  it('HTMLとスタイルが変わらないこと', async () => {
    const {asFragment} = render(<TimeText dateTime="2021-01-01T00:00:00.000Z" />);

    expect(asFragment()).toMatchSnapshot();
  });

  it('投稿日時がYYYY.MM.DDの形式で表示されていること', () => {
    const {getByText} = render(<TimeText dateTime="2021-01-01T00:00:00.000Z" />);

    expect(getByText('2021.01.01')).toBeInTheDocument();
  });
});
