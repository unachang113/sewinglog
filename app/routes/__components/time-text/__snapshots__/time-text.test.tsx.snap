// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TimeText > HTMLとスタイルが変わらないこと 1`] = `
<DocumentFragment>
  <time
    datetime="2021-01-01"
  >
    2021.01.01
  </time>
</DocumentFragment>
`;
